packages/fuel-indexer-tests/tests/e2e/indexing_postgres.rs

```diff
@@ -484,3 +484,82 @@ async fn test_index_metadata_is_saved_when_indexer_macro_is_called_postgres() {
     assert!(block_height >= 1);
     assert!(time >= 1);
 }
+
+#[actix_web::test]
+#[cfg(all(feature = "e2e", feature = "postgres"))]
+async fn test_index_respects_start_block_postgres() {
+    let pool = postgres_connection_pool().await;
+    let mut srvc = indexer_service_postgres().await;
+
+    let contract = connect_to_deployed_contract().await.unwrap();
+    let app = test::init_service(app(contract)).await;
+    let req = test::TestRequest::get().uri("/block_height").to_request();
+    let res = test::call_and_read_body(&app, req).await;
+    let block_height = String::from_utf8(res.to_vec())
+        .unwrap()
+        .parse::<u64>()
+        .unwrap();
+
+    let mut manifest: Manifest =
+        serde_yaml::from_str(assets::FUEL_INDEXER_TEST_MANIFEST).expect("Bad yaml file.");
+
+    manifest.start_block = Some(block_height + 2);
+
+    update_test_manifest_asset_paths(&mut manifest);
+
+    srvc.register_index_from_manifest(manifest)
+        .await
+        .expect("Failed to initialize indexer.");
+
+    sleep(Duration::from_secs(defaults::INDEXED_EVENT_WAIT)).await;
+
+    let mut conn = pool.acquire().await.unwrap();
+    let pre_check = sqlx::query(&format!(
+        "SELECT * FROM fuel_indexer_test.block where height = '{}'",
```

Comment:
Block height is a `u64`? Therefore doesn't require string formatting in the query? `height = '{}'` to `height = {}`